.circleci/config.yml

```
@@ -1,18 +1,18 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@4.8.1
+  architect: giantswarm/architect@4.21.0
 
 workflows:
   package-and-push-chart-on-tag:
     jobs:
       - architect/push-to-app-catalog:
           context: "architect"
-          # executor: "app-build-suite" # uncomment this if you want automatic metadata generation and helm chart linting
-          name: "package and push {APP-NAME} chart"
-          app_catalog: "giantswarm-playground-catalog"
-          app_catalog_test: "giantswarm-playground-test-catalog"
-          chart: "{APP-NAME}"
+          executor: "app-build-suite"
+          name: "push-to-default-app-catalog"
+          app_catalog: "default-catalog"
+          app_catalog_test: "default-test-catalog"
```

As we don't need the app in the default catalog, it's enough to have it in the control-plane catalog only

```suggestion
          name: "push-to-control-plane-app-catalog"
          app_catalog: "control-plane-catalog"
          app_catalog_test: "control-plane-test-catalog"
```